### PR TITLE
modem+hydrogen scripts

### PR DESCRIPTION
**Pull request by laurencooney**
Migrated these from assembly-station and modem_signal_checker repos for use on the fishbowl laptop. 

- hydrogen script should look largely the same
- modem test is targeted to just parse the internal_sbd.txt and assumes a network connection to the drone (most likely dronenet-mobile)

For ease of use by fishbowl folks, I've added the source command for the virtual environment and aliases to the scripts (`end_modem` and `hydrogen`) in the fishbowlmacbook .bashrc.
If these get used frequently, we should add to devpi and do it proper.

### CODE REVIEW

File: modem/MODEM.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# modem signal test
+
+Runs analysis on Iridium SBD signal quality from modem data on a baseboard. 
+
+##Procedure
+1. Baseboard is powered on for first time outside. No need to do anything after power on, Iridium will start collecting data immediately.
+2. Baseboard sits outside for > 30 mins, ideally between 1-1.5 hours
+3. Script operator brings laptop, bullet box, and serial cable to baseboard outside
+4. Power on bullet box and connect laptop to dronenet-mobile
```
Comment (Dillonmcewan) [inline modem/MODEM.md]: FYI we could do this over an ethernet cable pretty easily if the bullet box is a hassle

File: modem/standalone_test_end.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -x
+
+export TIMESTAMP=$(date -Isec)
+echo "export TIMESTAMP=$TIMESTAMP" > /root/latest_timestamp
```
Comment (Dillonmcewan) [inline modem/standalone_test_end.sh]: Nit: `TIMESTAMP` is pretty generic. Could conflict with other env vars

File: modem/standalone_test_end.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+mkdir -p "/root/test/${TIMESTAMP}"
+
+echo "$1" > "/root/test/${TIMESTAMP}/notes.txt"
+
+source /root/latest_timestamp
+send_pequod_cmd --cmd flush_data_cmd
+sleep 3
+cp /data/log/latest/modem/*state* /root/test/${TIMESTAMP}/
+cp /data/log/latest/modem-stdout.trace /root/test/${TIMESTAMP}/stdout.txt
+if [ ! "$(ls /root/test/${TIMESTAMP}/*state*)" = "" ]; then
```
Comment (Dillonmcewan) [inline modem/standalone_test_end.sh]: Could you add a comment on why this is needed?

Comment (laurencooney) [reply modem/standalone_test_end.sh]: Not sure if this is referring to creating the test directories or removing the previous internal_sbd file.
- test directories have been useful for organization, added a comment
- deleting old internal_sbd is no longer relevant as we are creating at test end, removed

File: modem/MODEM.md (hunk 1 above)
Comment (laurencooney) [reply modem/MODEM.md]: @Dillonmcewan ah that would be great. I'll create a separate ticket for this

**Pull request merged**